types/gburnett__draft-js/tslint.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Why do you need to turn those rules off? It's preferred to follow the standard style on DT, or add inline annotations to the places where a rule must be broken.